app/assets/stylesheets/base.scss

```diff
@@ -0,0 +1,9 @@
+.row {
```

Comment:
Sounds good. Gives us a bit more specificity by keeping us from adding too much onto the bootstrap classes.